tests/client_test.py

```diff
@@ -1068,7 +1068,8 @@ class CallbackException(Exception):
 
         def cb(room, event):
             if isinstance(event, RoomMemberEvent):
-                raise CallbackException()
+                if instance(room, str):
+                    raise CallbackException()
```

Comment:
Based on the test failures, you'll need to fix this + run `black` formatting.
```suggestion
                if isinstance(room, str):
                    raise CallbackException()
```
Though honestly, I'd also suggest combining this with the line above, to be:
`if isinstance(event, RoomMemberEvent) or isinstance(room, str):